### ISSUE TITLE
Genius-f logs for NER

### ISSUE THREAD

**Issue by Sreyan88** [beyondguo/genius#12]
Hi there! Great repo. Would you mind sharing the logs for Genius-f fine-tuning for the NER use case? Or would you mind giving an estimate of the final rouge metrics and how much time it took to converge?

Thank You!

**Comment by beyondguo** [comment]
Hi, thanks for your interest! For the NER task, we directly use the pre-trained `genius-large` model, without fine-tuning. If you want to fine-tune the model for NER, you can modify the [finetuning script for classification task](https://github.com/beyondguo/genius/blob/master/augmentation_clf/genius_finetune.py). I didn't remember the concrete training time (maybe several minutes for hendreds of samples?)

**Comment by Sreyan88** [comment]
I try to fine-tune with Genius-f on a custom NER dataset and it seems to converge in about 2 epochs (lowest eval loss). Is this behavior normal?
